src/CMakeLists.txt

```diff
@@ -781,7 +781,7 @@ endif()
 
 if(MSVC)
 	# Visual Studio
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /D _WIN32_WINNT=0x0601 /D WIN32_LEAN_AND_MEAN")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /D _WIN32_WINNT=0x0601 /D WIN32_LEAN_AND_MEAN /D CRT_SECURE_NO_WARNINGS")
```

Comment:
missing `_`?

Reply:
tis, not sure why I got no warnings. Maybe didn't need it for the main minetest target in the first place?